src/styles/main.scss

```
@@ -708,3 +681,73 @@ Fixing styling for the code example tables in the "Migrating From..." pages
   top: calc(50% - 11px);
   left: 18px;
 }
+
+// CSS styling for the date modifiex X ago
+.quick-info-list {
+  background: #FFFFFF;
+  border: 1px solid #DDDDDD;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  display: flex;
+}
+
+#time-posted-on {
+  padding: 10px 10px 10px 10px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  color: #343434;
+}
+
+.quick-info-list img {
+  margin: 0px 3px 3px 0px;
+}
+
+// badge tags
+.server-version-badge {
+  background-color: white;
+  border: none;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 18px;
+  padding: 0px;
+  cursor: pointer;
+
+  &:not(:nth-child(1)):not(:nth-child(2)):before {
+    content:"\A";
+    width: 3px;
+    height: 3px;
+    border-radius:50%;
+    background: #343434;
+    display:inline-block;
+    margin: 0px 5px 3px 0px;
+  }
+}
+
+.server-version-wrapper {
+  padding: 10px;
+  flex-grow: 1;
+  border-left: 1px solid #DDDDDD;
+}
+
+#tooltiptime {
```

I think we usually use kebab-case for our css.
```suggestion
#tooltip-time {
```